src/cls/IPM/Main.cls

```diff
@@ -960,12 +961,16 @@ ClassMethod GetListModules(pNamespace As %String = {$Namespace}, pSearch As %Str
         Set tParams = tParams _ $ListBuild("?")
         Set tArgs($Increment(tArgs)) = tItem
       }
-      Set tQuery = tQuery _ " WHERE name IN (" _ $ListToString(tParams, ", ") _ ")"
+      Set tQuery = tQuery _ " AND name IN (" _ $ListToString(tParams, ", ") _ ")"
     } ElseIf pSearch'="*" {
-      Set tQuery = tQuery _ " WHERE name " _ $Select(pSearch["*": "like ?", 1: "= ?")
+      Set tQuery = tQuery _ " AND name " _ $Select(pSearch["*": "like ?", 1: "= ?")
       Set tArgs($Increment(tArgs)) = $Translate(pSearch, "*", "%")
     }
   }
+  If pRepository'="" {
+	Set tQuery = tQuery _ " AND Repository = ?"
```

Comment:
Nit: Fix spacing of contents inside if block

Reply:
Done.